facebookads/objects.py

```diff
@@ -487,6 +488,9 @@ def remote_create(
         params = {} if not params else params.copy()
         params.update(self.export_data())
 
+        # some private endpoints require a custom relative_path
+        relative_path = relative_path or (self.get_parent_id_assured(), self.get_endpoint())
```

Comment:
we still pass releative_path for remote_create so that we hit the <buisness_id>/adaccount enpoint and when this is none we still want to use the regular method for generating this argument to pass to batch.add or api.call